Draft: Proposal for marketing working group

thibaudcolas opened the pull request:
[View the proposed charter](https://github.com/thibaudcolas/dsf-working-groups/blob/marketing-wg/active/marketing.md). This is a draft charter written by @sabderemane and I for a new Marketing working group for Django, in charge of both strategy and execution. It’s inspired by our website and social media working groups, though this is a bit more complex as marketing overlaps with a lot of existing teams’ remit. 

- Marketing work has to be coordinated with a lot of other aspects of the project / foundation
- It needs a lot of strategic thinking at times, and a lot of just pure execution at other times
- Lots of the execution relies on having access to relatively sensitive systems (website publishing, social media)
- The execution also relies on the group having enough autonomy when possible, and enough stakeholder review when needed

## Proposal status

The draft attempts to balance all of the above, hopefully making it clear enough when we can have this group be autonomous and when to otherwise coordinate. We need more people with more capacity than @sabderemane and I to take this further. We thought creating the draft might help.

thibaudcolas committed:
Add first draft marketing WG proposal

tim-schilling commented on active/marketing.md:
```diff
@@ -0,0 +1,78 @@
+
+## Budget
+
+No allocated budget. The group will likely request a budget to hire external marketing experts in the future.
```
May be helpful to also include tools as a possible expenditure. I'm thinking something like figma

thibaudcolas replied:
I expect this will be a must, but I’m a bit hesitant to define this in the initial charter as people on the group might have opinions?

tim-schilling replied:
~~I don't think it should call out Figma, but it can call out that the WG can spend money on tools. Or that this can be revisited on a as needed basis.~~

The no allocated budget line with an exception for hiring a consultant reads that there's no budget without a governance change. I guess I'm suggesting it be flexible.

tim-schilling commented:
This looks great! It's generic enough to get people moving, but provides the necessary specifics. The only other minor thing would be unless the board is requiring kpis, I think that could be left to the team to decide to implement.

thibaudcolas committed:
Small tweaks

thibaudcolas commented:
Got quite good feedback privately so far, so I think this is ready for wider review, still some rough edges here and there.

czue commented on active/marketing.md:
```diff
@@ -0,0 +1,81 @@
+
+- Management of fundraising pages on [djangoproject.com](http://djangoproject.com), in collaboration with the fundraising WG.
+- Contributions of marketing content to the "docs" [djangoproject.com](http://djangoproject.com) website content, in collaboration with Django core contributors.
+- Publication of new content across Django online community and social media channels, in collaboration with the Online Community WG and Social media WG.
```
Is there a chance we would want to have the social media WG absorbed/disbanded once the marketing WG was in place? Unlike website and online community, it feels like social media is ~entirely a subset of marketing.

benjaoming replied:
I think it's better to have individual sub-WGs with clear-cut work. So the Social Media working group would continue, but with a more well-defined task thanks to the parent Marketing WG.

I don't mind that Marketing WG makes its own executive calls on Social Media posts.

thibaudcolas replied:
Yep we have discussed this heaps. In the end I feel like there’s plenty enough to do, that we could use 5-10 volunteers focused on social media, and 10 or so on other marketing activities. It felt to me like smaller groups would work better than one big one.

Day to day, I’d hope lots of discussions are held as one big group with everyone involved in this space rather than siloed per group. It’d largely just be the "execution" that is kept separate.

czue reviewed (commented):
Looks useful! One thing that might be helpful is saying more about the expectations for membership. E.g. what types of hard skills would be useful (I assume writing and design would feature) as well as the expectations around time commitement for involvement. Given it sounds like a serious effort, we might want to filter for serious people.

carltongibson reviewed (commented):
Hey @thibaudcolas @sabderemane — this is good. 

As I read it, I half-feel — "half"! — that we're missing a mention of historical challenges. Namely, many times folks have tried to get moving on this and failed for lack of ability to _Get to Yes_, lining up all the stake holders, especially combined with the rolling nature of the Board membership. (i.e. Just as someone is making progress, or getting close, their term expires, they leave the Board, and even **if** they can hand off, there's a loss of momentum which kills the efforts.) 

I'm totally happy with what's written here — delegated powers etc. But we block ourselves, through lack of clarity on _why_ things take ages in Django-land. A specific, _this group's whole purpose is to avoid that problem_ would help have something to point to, and might shake out a few more points worth making. 🤷

I did say I'd be SC liaison to begin, so you can fill me in there if you like :)

benjaoming commented on active/marketing.md:
```diff
@@ -0,0 +1,81 @@
+
+## Scope of responsibilities
+
+The working group is responsible for planning and execution of marketing activities, and for advising the DSF Board on strategic decisions related to marketing. The goal of those marketing activities is to:
```
```suggestion
The working group is responsible for:

* Planning and execution of marketing activities
* Proposing new strategies for marketing activities
* Advising the DSF Board on strategic decisions related to marketing and strategic communication

The goals of those marketing activities are:
```

benjaoming replied:
I don't think I captured it perfectly above, but I wanted the 3 points above to convey:

* (responsible+mandate for implementing marketing strategies approved by DSF board)
* (responsible for proposing NEW marketing strategies that DSF board approves)
* (advising DSF on marketing and strategic communication)

We could also try to define a bit more what a "marketing activity" vs a "marketing strategy" is. I guess no one is really confused, so maybe that would just be fluff :shrug:

benjaoming replied:
I tried to squeeze in something about "strategic communication"... I dunno, but to sort of counter-balance the mindset of commercial marketing? This can maybe lead to some discussions about "quality over quantity" in the KPIs that the marketing would orient towards.

We're asking ourselves really deep questions about what the Django brand is, why it's a trusth-worthy technology and community, and what sort of messaging and activities can keep us there on the long term. At the end of the day, most activities are a sort of marketing, whether that's intentional or not :slightly_smiling_face:

Example: Blog posts (without any sense of marketing messages nor intentions) from the Steering Council about Django's future are actually great marketing :smile:

thibaudcolas replied:
I like that!

benjaoming commented on active/marketing.md:
```diff
@@ -0,0 +1,81 @@
+The working group is responsible for planning and execution of marketing activities, and for advising the DSF Board on strategic decisions related to marketing. The goal of those marketing activities is to:
+
+- Promote Django: to increase awareness, understanding, and adoption of the framework. And beyond the framework, to strengthen the public narrative about the project, its community, and the Django Software Foundation.
+- Ensure that Django is represented clearly and positively across all channels that are relevant to our audiences (such as developers, organisations, educators, open source contributors, etc).
```
```suggestion
- Ensure that Django is represented clearly and positively across all channels that are relevant to our audiences (such as developers, organisations, educators, open source contributors, etc).
- Increasing strategic coherence of activities across our communication channels
```

thibaudcolas replied:
Interesting. I like it, wonder what this will look like in practice.

benjaoming commented on active/marketing.md:
```diff
@@ -0,0 +1,81 @@
+
+Delegated responsibilities for the group to work on autonomously:
+
+- Management of the "www" djangoproject.com website content, in particular the Django blog and other content pages, excluding fundraising pages.
```
```suggestion
- Management of the "www" djangoproject.com website content, in particular the Django blog and other channels that are closely related to marketing.
```

I would just remove it - I know there is a Fundraising WG. But maybe we can live with the overlap, rather than listing it out? If the Marketing WG is the umbrella of sub-WG that executes on specific channels, then the Marketing WG should simply always be responsible for not interfering... or maybe it would do so with good reason 🤷

ulgens replied:
Wouldn't it be better if it were a joint responsibility?

benjaoming replied:
I don't believe much in joint responsibility :smile: (it often ends up in no responsibility)

The issue is the same for Website and Social Media WGs..

For the blog content, I think we're trying to find a solution for a specific part of the website with no clear ownership.

Other than that, _the entire website is marketing_, if framed the right way.

I think spelling out something specific will be tricky. Always good to have clarity, but not always possible. I think that clarity is better found in aiming the group as an umbrella marketing group that does guidelines, strategies, plans, sometimes content when it's vulnerable to marketing considerations.. but **mainly to cut out the work for other groups**. It should rarely intersect with their work, but maybe an overlap in individual group membership will make this more easy and obvious.

thibaudcolas replied:
I’m not a fan of the current wording, agree with the problem here, but not sure I like this approach better. But clearly something to mull over. I singled out fundraising because it is meant to be the fundraising people being autonomous on this. But in practice it’s obviously not ideal when groups work in silo.

benjaoming reviewed (approved):
After pondering if the Fundraising WG had overlaps, I didn't really find any, and I think this looks great 👍 

Have some suggestions, that's all...

thibaudcolas commented:
Thanks all for the feedback, keep it coming :) And if you have ideas for things the group could do as well, particularly in the context of our [website redesign](https://github.com/django/djangoproject.com/issues/2287) but all other channels relevant too. The ideas might not change the charter but it’ll certainly be inspiring for others.

@carltongibson yep I think we do need that mention of historical / just structural challenges. In my opinion it will help that we treat this seriously enough to produce "marketing deliverables" (just docs really), that will help the knowledge transfer more easily. And I think it also helps that we’ve just grown the foundation side of Django a lot. It’s easier for new people to get involved with this kind of volunteering, and it’s easier to retain people who want to take a step back but stay involved in advisory capacity. Will mull over how to write this.

vossisboss commented:
Hey y'all. Just wanted to chime in and say that I would happy to be a Wagtail liason for this group along with Thibaud. There is plenty we already have to do for Wagtail but there will definitely be opportunities for us to join forces and I would like to be tuned in to this group's activities so we can identify and shape those opportunities together.

thibaudcolas committed:
Apply suggestions from code review

Co-authored-by: Benjamin Balder Bach <benjaoming@gmail.com>

thibaudcolas commented:
👋 I’ve updated the proposal title and added a disclaimer that @sabderemane aren’t super confident in taking this further, just due to the amount of work:

>  […] We need more people with more capacity than @sabderemane and I to take this further, and thought creating the draft might help.

And merged feedback from @benjaoming as-is. Still keen to get more feedback! From DjangoCon Europe discussions, the fundamental issues we discussed that need addressing are:

1. Volunteer capacity too low for marketing workload
2. What we should do as free volunteer work vs. paid work
3. Separate needs for a strategy vs. execution
